interface/modules/custom_modules/oe-module-claimrev-connect/public/setup.php

```diff
@@ -0,0 +1,165 @@
+                        </h6>
+                        <?php 
+                            if(ClaimRevModuleSetup::CouldSftpServiceCauseIssues())  
+                            {
+                        ?>                                
+                                The SFTP service is still activated to send claims. 
+                                We have noticed that this service can cause our service not to work correctly. 
+                                If you would like to deactivate it, click the following button. 
+                                Note: if you're sending claims elsewhere through SFTP, this would stop that.
+                                <form method="post" action="<?=$_SERVER['PHP_SELF'];?>">
+                                    <button type="submit" name="deactivateSftp" class="btn btn-primary">Deactivate</button>
+                                </form>
+                        <?php
+                            }
+                            else
+                            {
+                        ?>
+                                The SFTP Service has been disabled, this is good and will prevent the service from working against sending your claims.
+                                However if you would like to reactivate it then click this button.
+                                <form method="post" action="<?=$_SERVER['PHP_SELF'];?>">
+                                    <button type="submit" name="reactivateSftp" class="btn btn-primary">Reactivate</button>
+                                </form>
+                        <?php
+
+                            }
+                        ?>
+                    </li>
+                </ul>
+             
+                </div>
+        </div>
+    </div>
+    <div class="row"> 
+        <div class="col">
+            <h1>Background Services</h1>        
+        </div>
+    </div>
+    <div class="row">
+        <div class="col">
+            <div class="card">
+                <table class="table">
+                    <thead>
+                        <tr>
+                            <th scope="col">Name</th>
+                            <th scope="col">Active</th>
+                            <th scope="col">Running</th>
+                            <th scope="col">Next Run</th>
+                            <th scope="col">Execute Interval</th>   
+                            <th scope="col">Function</th>       
+                            <th scope="col">Require Once</th>          
+                        </tr>
+                    </thead>
+                    <tbody>
+                            <?php 
+                                foreach($services as $service)
+                                {
+                            ?>
+                                <tr>
+                                    <td>
+                                        <?php echo($service["name"])  ?> - <?php echo($service["title"]) ?>
```

Comment:
text() above line and lines 138, 141, 144, 147, 150, 153

Reply:
done